applications/luci-app-acme/htdocs/luci-static/resources/view/acme.js

```diff
@@ -524,15 +524,17 @@ return view.extend({
 		};
 
 		o = s.taboption('advanced', form.Value, "acme_server", _("ACME server URL"),
-			_('Use a custom CA.') +	' ' + _('Custom ACME server directory URL.'));
+			_('Use a custom CA.') +	' ' + _('Custom ACME server directory URL.') + '<br />' +
+			'<a href="https://github.com/acmesh-official/acme.sh/wiki/Server" target="_blank">' + _('See more') + '</a>'
```

Comment:
Notionally, the UI is independent of underlying implementation (i.e., it could be something other than acme.sh), so we should link to a generic documentation of the ACME protocol if we are linking to anything. Or just drop this...